packages/@aws-cdk/aws-config/lib/rule.ts

```diff
@@ -1636,8 +1658,10 @@ export class ResourceType {
   public static readonly BACKUP_BACKUP_SELECTION = new ResourceType('AWS::Backup::BackupSelection');
   /** AWS Backup backup vault */
   public static readonly BACKUP_BACKUP_VAULT = new ResourceType('AWS::Backup::BackupVault');
-  /** AWS Backup backup recovery point */
```

Comment:
nice catch. It's funny to me how many of these silly doc errors exist in cdk lol